_pages/05-tests.md

```diff
@@ -2,64 +2,64 @@
 layout: page
 permalink: /tests/
 title: "Tests"
+page-title: "M-Lab Tests"
 menu-item: true
-grid-quick-links: true
+breadcrumb: tests
 grid-section:
-  - group-heading: "Performance"
+  - group-heading: "Current Tests"
     group-description: "Performance tests measure basic and advanced performance characteristics of your network, such as speed, latency, jitter, and much more."
     group-list-class: "l-grid l-3col"
     grid-items:
       - link: "/tests/ndt/"
         image: "ndt.png"
         heading: "NDT (Network Diagnostic Tool)"
         desc: "Tests your connection speed, and provides a sophisticated diagnosis of problems limiting speed."
+        categories: "performance"
       - link: "/tests/npad/"
         image: "npad.png"
         heading: "NPAD (Network Path &amp; Application Diagnostics)"
         desc: "Diagnoses common problems that impact last-mile broadband networks."
+        categories: "performance"
       - link: "/tests/neubot/"
         image: "neubot.png"
         heading: "Neubot"
         desc: "Performs periodic tests to measure network performance and traffic throttling."
+        categories: "performance"
       - link: "/tests/bismark/"
         image: "neubot.png"
         heading: "BISmark"
         desc: "Allows you to apply to host a home router that also tests network performance over time."
+        categories: "performance"
       - link: "/tests/sidestream/"
         image: "sidestream.png"
         heading: "SideStream"
         desc: "Collects statistics about the TCP connections running on the M-Lab platform."
-  - group-heading: "Performance Mobile"
-    group-list-class: "l-grid l-3col"
-    grid-items:
+        categories: "performance"
       - link: "/tests/mobiperf/"
         image: "mobile.png"
         heading: "MobiPerf"
         desc: "Measures network performance on mobile platforms."
-  - group-heading: "Transparency"
-    group-description: "Transparency tests provide insight into network management practices, such as application-specific blocking, throttling, traffic shaping, and other practices that determine your ability to use some applications or to access some content on the Internet."
-    group-list-class: "l-grid l-3col"
-    grid-items:
+        categories: "performance"
       - link: "/tests/ooni/"
         image: "ooni.png"
         heading: "OONI Probe"
         desc: "Measures specific use cases of network interference."
+        categories: "transparency"
       - link: "/tests/neubot/"
         image: "neubot.png"
         heading: "Neubot"
```

Comment:
ah, good catch. multiple categories/tags should be doable, but this PR just has the old categories with the same looping as before, and I'd missed Neubot being in there twice. so we're seeing it twice because the page gets parsed by "category"

Proposed fix: remove the second category for Neubot for now, then re-implement tags on specific tests in a different way.